strings/tcg/set_one.json

```diff
@@ -45,7 +45,7 @@
 			"id": "changeling",
 			"name": "Armoured Changeling",
 			"desc": "The strange creatures known as changelings have been known to develop natural armour as a defense mechanism when in combat.",
-			"rules": "{$Changeling}",
+			"rules": "{$Changeling} {$Deadeye}: Add 1 creature with Changeling from your Deck to your hand.",
```

Comment:
It's implied, but either the rules or the specific card effect should specify a shuffle following pulling cards from your deck.

Reply:
Will add so to the rules